e-shop-manager/src/main/java/it/unipd/mtss/business/BillImpl.java

```diff
@@ -20,6 +20,10 @@ public double getOrderPrice(List<EItem> itemsOrdered, User user)
         double total = getTotal(itemsOrdered);
         double discount = 0;
         
+        if(itemsOrdered.size() > 30){
+            throw new BillException();
```

Comment:
Non è il caso di modificare l'eccezione in modo da potergli mandare il messaggio d'errore che si vuole produrre?